django/db/transaction.py

```diff
@@ -296,14 +305,14 @@ def __exit__(self, exc_type, exc_value, traceback):
                     connection.in_atomic_block = False
 
 
-def atomic(using=None, savepoint=True, durable=False):
+def atomic(using=None, savepoint=True, durable=False, _test_marker=False):
```

Comment:
Probably not desirable to expose private arguments on public interfaces like this.